2.9.0/jiva-operator.yaml

```
@@ -0,0 +1,4037 @@
+metadata:
+  creationTimestamp: null
+  name: jiva-operator
+  namespace: openebs
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - services/finalizers
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - '*'
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - '*'
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - apps
+  resourceNames:
+  - jiva-operator
+  resources:
+  - deployments/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - apps
+  resources:
+  - replicasets
+  verbs:
+  - get
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - '*'
+- apiGroups:
+  - openebs.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: jiva-operator
+  namespace: openebs
+subjects:
+- kind: ServiceAccount
+  name: jiva-operator
+roleRef:
+  kind: Role
+  name: jiva-operator
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jiva-operator
+  namespace: openebs
+  labels:
+    openebs.io/component-name: jiva-operator
+    openebs.io/version: 2.9.0
+    name: jiva-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: jiva-operator
+  template:
+    metadata:
+      labels:
+        name: jiva-operator
+        openebs.io/component-name: jiva-operator
+        openebs.io/version: 2.9.0
+    spec:
+      serviceAccountName: jiva-operator
+      containers:
+        - name: jiva-operator
+          # Replace this with the built image name
+          image: openebs/jiva-operator:2.9.0
+          command:
+          - jiva-operator
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: OPENEBS_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "jiva-operator"
+            - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE
+              value: "openebs/jiva:2.9.0"
+            - name: OPENEBS_IO_JIVA_REPLICA_IMAGE
+              value: "openebs/jiva:2.9.0"
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8282
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8282
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          resources:
+            limits:
+              cpu: 100m
+              memory: 30Mi
+            requests:
+              cpu: 100m
+              memory: 20Mi
```

Can we increase the memory limit
```suggestion
          resources:
            limits:
              cpu: 100m
              memory: 100Mi
            requests:
              cpu: 100m
              memory: 100Mi
```

---

limit and request both get same , we can have `requests.memory: 50Mi `?